include/Core/EmailLogger.php

```diff
@@ -64,6 +64,12 @@ public function log_email( $original_mail_info ) {
 		 *
 		 * @since 2.0.0
 		 */
+		// if this is being called from import addon
```

Comment:
@raaveeraa This approach is perfect.

But instead of declaring new variables, you can set it as keys to the default array that is passed to the `wp_parse_args` function.

Reply:
Got it @sudar . This has been taken care.